src/common/KVStore/IKVStore.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef _IKVSTORE_
+#define _IKVSTORE_
+
+namespace snail{
+	
+class MockKVStore_result_type{
+public:
+	MockKVStore_result_type(bool b = false) : is_success_(b), value_("") {}
+	MockKVStore_result_type(bool b, std::string& value) : is_success_(b), value_(value){ }
+	MockKVStore_result_type(bool b, std::string&& value) : is_success_(b), value_(value){ }
+	virtual ~MockKVStore_result_type() = default;
+	
+	bool is_success()const {return is_success_;} 
```

Comment:
why do this 2 member functions named as different ways?

Reply:
sorry, I donot know what do you mean?